### ISSUE TITLE
Webmail: When OIDC is configured and OIDC provider fails, TMail fallback to basic auth

### ISSUE THREAD

**Issue by chibenwa** [linagora/tmail-flutter#2672]
### Description

GIVEN OIDC was either configured or detected (.well-known/webfinger)
WHEN the provider / auth / whatever fails
THEN tmail should not fallback to basic auth

It should only fallback to basic auth if OIDC is neither configured nor detected.

Instead we should display a page explaning what went wrong and allowing to retry.

### Current behavior

With some OIDC error Twake mail fallbacks to basic auth.

Cc @guimard

**Comment by hoangdat** [comment]
> With some OIDC error Twake mail fallbacks to basic auth.

No. only request `/.well-known/webfinger` have error (4XX or CORS). If we had OIDC configuration, no way to back to basic auth

**Comment by chibenwa** [comment]
I will keep this around as I did remember it happening to me...

> No. only request /.well-known/webfinger have error (4XX or CORS). If we had OIDC configuration, no way to back to basic auth

I'm not 100% sure of that assertion...
